Source/Projections/Builder/ProjectionBuilder.cs

```diff
@@ -49,9 +53,11 @@ public ProjectionMethodsForReadModelBuilder<TReadModel> ForReadModel<TReadModel>
         {
             if (_methodsBuilder != default)
             {
-                // Throw exceptionee c
+                var genericType = _methodsBuilder.GetType().GetGenericArguments()[0];
+                throw new ReadModelAlreadyDefinedForProjection(_projectionId, _scopeId, genericType);
```

Comment:
This feels very sneaky. I remember we also kept the type as a property in this class in TS - feels like a valuable thing to do here as well to make this a bit easier on the eyes :)

Reply:
A little sneaky. I guess that we can just use the typeof(TReadModel) here I guess